permits/templates/permits/permit_request_detail.html

```diff
@@ -224,11 +216,25 @@ <h2>{% trans "Données de base" %}</h2>
       </a>
     </dd>
   {% endif %}
-  {% if config.ENABLE_GEOCALENDAR %}
-  <dt>{% trans "Publié" %}</dt>
-  <dd>
-    {{permit_request.is_public|yesno:"Oui,Non" }}
-  </dd>
+  {% if history %}
+    <div id="history_timeline">
+      <h4><b>{% trans "Suivi" %}</b></h4>
+      <br>
+      <ul>
+        {% for archive in history %}
+          {% if archive.status == permit_request.STATUS_PROCESSING and forloop.last %}
+            <li><i class="fa fa-circle status{{archive.status}}"></i> {{ archive.history_date }} - <b>{% trans "Demande soumise par " %}</b>{{ archive.history_user }}</li>
+          {% else %}
+            <li><i class="fa fa-circle status{{archive.status}}"></i> {{ archive.history_date }} - <b>{{ archive.get_status_display }}</b> - {{ archive.history_user }}</li>
```

Comment:
```suggestion
            <li><i class="fa fa-circle status{{archive.status}}"></i> {{ archive.history_date }} - <b>{{ archive.get_status_display }}</b> ({{ archive.history_user }})</li>
```

It's personnal but I find it more readable with the `(xxx)`